packages/jest-jasmine2/src/types.ts

```diff
@@ -48,8 +48,8 @@ export interface Spy extends Record<string, any> {
 
 type JasmineMatcher = {
   (matchersUtil: unknown, context: unknown): JasmineMatcher;
-  compare: () => RawMatcherFn;
-  negativeCompare: () => RawMatcherFn;
+  compare(...args: Array<unknown>): unknown;
+  negativeCompare(...args: Array<unknown>): unknown;
```

Comment:
These type were ignored with `// @ts-expect-error`. Perhaps it is better idea to rework them into `unknown`s and remove `// @ts-expect-error`?

Reply:
yeah, I think so